x-pack/legacy/plugins/siem/public/pages/network/network.tsx

```diff
@@ -123,167 +110,42 @@ const NetworkComponent = React.memo<NetworkComponentProps>(
-                                false,
-                                'showMorePagesIndicator',
-                                pageInfo
-                              )}
-                              totalCount={totalCount}
-                              type={networkModel.NetworkType.page}
-                            />
-                          )}
-                        </NetworkTopNFlowQuery>
-                      </EuiFlexItem>
-
-                      <EuiFlexItem>
-                        <NetworkTopNFlowQuery
-                          endDate={to}
-                          flowTarget={FlowTargetNew.destination}
-                          filterQuery={filterQuery}
-                          skip={isInitializing}
-                          sourceId="default"
-                          startDate={from}
-                          type={networkModel.NetworkType.page}
-                        >
-                          {({
-                            id,
-                            inspect,
-                            isInspected,
-                            loading,
-                            loadPage,
-                            networkTopNFlow,
-                            pageInfo,
-                            refetch,
-                            totalCount,
-                          }) => (
-                            <NetworkTopNFlowTableManage
-                              data={networkTopNFlow}
-                              fakeTotalCount={getOr(50, 'fakeTotalCount', pageInfo)}
-                              flowTargeted={FlowTargetNew.destination}
-                              id={id}
-                              indexPattern={indexPattern}
-                              inspect={inspect}
-                              isInspect={isInspected}
-                              loading={loading}
-                              loadPage={loadPage}
-                              refetch={refetch}
-                              setQuery={setQuery}
-                              showMorePagesIndicator={getOr(
-                                false,
-                                'showMorePagesIndicator',
-                                pageInfo
-                              )}
-                              totalCount={totalCount}
-                              type={networkModel.NetworkType.page}
-                            />
-                          )}
-                        </NetworkTopNFlowQuery>
-                      </EuiFlexItem>
-                    </ConditionalFlexGroup>
+                    <SiemNavigation
+                      navTabs={navTabsNetwork(hasMlUserPermissions(capabilities))}
+                      display={sourceId}
+                      showBorder={true}
+                    />
 
                     <EuiSpacer />
 
-                    <NetworkDnsQuery
-                      endDate={to}
+                    <NetworkTabs
```

Comment:
Since that we've got a `SiemNavigation` component, what do you think if we rename this to something like `NetworkTabsRoutes`?